project-base/app/tests/FrontendApiBundle/Test/GraphQlTestCase.php

```diff
@@ -388,4 +388,22 @@ private function createUploadedFiles(array $filePaths): array
 
         return $uploadedFiles;
     }
+
+    /**
+     * @param array $data
+     * @return array
+     */
+    protected function resolveReferenceDataAccessors(array $data): array
+    {
+        array_walk_recursive($data, function (&$value) {
+            if ($value instanceof ReferenceDataAccessor) {
+                $entity = $this->getReference($value->reference);
```

Comment:
I have an idea to make it a bit more universal so we are able to use it for domain references as well - I would add a third nullable property `$domainId` to `ReferenceDataAccessor` - if it is not null, `$this->getReferenceForDomain()` should be used here instead

Reply:
Added